plugins/PostProcessingPlugin/scripts/SearchAndReplace.py

```diff
@@ -23,37 +23,170 @@ def getSettingDataString(self):
             {
                 "search":
                 {
-                    "label": "Search",
-                    "description": "All occurrences of this text will get replaced by the replacement text.",
+                    "label": "Search for:",
+                    "description": "CASE SPECIFIC.  'LAYER' is not the same as 'Layer'.  All occurrences of this text (within the search range) will be replaced by the 'Replace with' string.",
                     "type": "str",
                     "default_value": ""
                 },
                 "replace":
                 {
-                    "label": "Replace",
-                    "description": "The search text will get replaced by this text.",
+                    "label": "Replace with:",
+                    "description": "The 'Search For' text will get replaced by this text.  For MultiLine insertions use the newline character 'backslash plus n' as the delimiter. If your Search term ends with a 'newline' remember to add 'newline' to the end of this Replace term.",
                     "type": "str",
                     "default_value": ""
                 },
                 "is_regex":
                 {
                     "label": "Use Regular Expressions",
-                    "description": "When enabled, the search text will be interpreted as a regular expression.",
+                    "description": "When disabled the search string is treated as a simple text string.  When enabled, the search text will be recompiled as a 'regular' python expression.",
```

Comment:
I don't understand what you mean with `'regular' python expression`. The text is actually interpreted as what is truely called a 'regular expression', so I find the previous text more explicit (with the enabled/disable addition)

Reply:
If anything other than `regular expression`, it should be `Python regular expression`; there are minute differences in how Python, PERL, Javascript interpret regular expressions.

Reply:
Yeah true, that would be the most accurate naming